README.md

```diff
@@ -110,89 +152,65 @@ execute the following in any OCaml top-level:
 ```ocaml
 # #use "topfind";;
 # #require "bap.top";;
-# open Bap.Std;;
 ```
 
-And everything should work just out of box, i.e. it will load all the
-dependencies, install top-level printers, etc.
+## Shell
 
+The BAP main frontend is a command line utility called
+[bap](https://github.com/BinaryAnalysisPlatform/bap-python). You can
+use it to explore the binary, run existing analysis, plugin your own
+behavior, load traces, and many more.
 
-## Using from shell
+To dump a program in various formats use `--dump` option (or its short
+equivalent `-d`). By default it dumps in the IR, but you can choose
+from various other formats. Use `--list-formats` option, to get the
+list. Note, however, that formats are added by plugins, and there are
```

Comment:
Runon sentence. Possible rewording:

However, formats are provided by plugins, so just because you don't see your preferred format listed doesn't mean you can't generate it. Check `opam` for plugins which may provide the format you want to read (the `bap-piqi` plugin provides protobuf, xml, and json, which cover many use cases).